backend/pyproject.toml

```diff
@@ -93,87 +98,87 @@ target-version = ['py310']
 [tool.ruff]
 target-version = "py311"
 line-length = 88
-src = ["src"]
+src = ["api", "tests", "migrations"]
 select = [
-    # "A", # flake8-builtins
```

Comment:
Changing formatting and content makes it really hard to review. Can you:
- if format is incorrect, please open a ticket on bootstrap so we'll fix.
- list what you changed and why.
- I see you've removed `flake8-builtins`. Sounds large; can't it be excludes of specific rules?

Reply:
Meant the ClassVar

Reply:
Yes, I think I need to open a ticket on the bootstrap.
But I will also make some comments here:

If we open the `flake8-builtins`, in the `nautilus-webui` `Ruff` will complain:
```
api/database/models.py:54:5: A003 Class attribute `id` is shadowing a Python builtin
api/database/models.py:73:5: A003 Class attribute `id` is shadowing a Python builtin
api/database/models.py:96:5: A003 Class attribute `id` is shadowing a Python builtin
api/database/models.py:107:5: A003 Class attribute `hash` is shadowing a Python builtin
api/database/models.py:109:5: A003 Class attribute `type` is shadowing a Python builtin
api/database/models.py:122:5: A003 Class attribute `id` is shadowing a Python builtin
api/routes/projects.py:23:5: A003 Class attribute `id` is shadowing a Python builtin
api/routes/users.py:16:5: A003 Class attribute `id` is shadowing a Python builtin
```
If you check the [document](https://beta.ruff.rs/docs/rules/builtin-attribute-shadowing/), it refers a link to the [stackoverflow](https://stackoverflow.com/questions/9109333/is-it-bad-practice-to-use-a-built-in-function-name-as-an-attribute-or-method-ide) question:
> It won't confuse the interpreter but it may confuse people reading your code. Unnecessary use of builtin names for attributes and methods should be avoided.
>
>Another ill-effect is that shadowing builtins confuses syntax highlighters in most python-aware editors (vi, emacs, pydev, idle, etc.) Also, some of the lint tools will warn about this practice.

I think in our case, useing `id` will not cause any confuses, because we just use it in the model class.

> Sounds large; can't it be excludes of specific rules?
Yes, you are right. We just need to exclude the `A003` not the `A`.